connexion/operations/openapi.py

```diff
@@ -273,6 +273,9 @@ def _get_body_argument(self, body, arguments, has_kwargs, sanitize):
 
         if x_body_name in arguments or has_kwargs:
             return {x_body_name: res}
+        # If the content type is "multipart/form-data" return the properties as-is
+        if self.consumes[0] == 'multipart/form-data':
```

Comment:
I believe it might be better to use this code. `connexion` maintains own list of what it understands as multipart.
```suggestion
        if self.consumes[0] in http_facts.FORM_CONTENT_TYPES:
```

Reply:
Ah, that's definitely better than mine :-)

Reply:
> 
> 
> I believe it might be better to use this code. `connexion` maintains own list of what it understands as multipart.
> 
> PS. I am not sure if amounts of dots is OK, but there is for sure a module `http_facts` in root package.

Not sure about just checking the FORM_CONTENTS_TYPES if one specifically needs to check for 'multipart/form-data'?